pkg/validate/security_context.go

```diff
@@ -616,3 +561,54 @@ func createCapabilityContainer(rc internalapi.RuntimeService, ic internalapi.Ima
 
 	return framework.CreateContainer(rc, ic, containerConfig, podID, podConfig)
 }
+
+func createAndCheckHostNetwork(rc internalapi.RuntimeService, ic internalapi.ImageManagerService, podSandboxName, hostNetworkPort string, hostNetwork bool) (podID string) {
+	By(fmt.Sprintf("creating a podSandbox with hostNetwork %v", hostNetwork))
+	podSandboxNamespace := &runtimeapi.NamespaceOption{
+		HostPid:     false,
+		HostIpc:     false,
+		HostNetwork: hostNetwork,
+	}
+	hostPath, podLogPath := createLogTempDir(podSandboxName)
+	podID, podConfig := createNamespacePodSandbox(rc, podSandboxNamespace, podSandboxName, podLogPath)
+
+	defer os.RemoveAll(hostPath) //clean up the TempDir
+
+	By("create a container in the sandbox")
+	command := []string{"sh", "-c", "netstat -ln"}
+	containerNamespace := &runtimeapi.NamespaceOption{
```

Comment:
We can remove `containerNamespace` and rename `podSandboxNamespace`. Since they have same value.